From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <jan.steffens@gmail.com>
Date: Sun, 16 Mar 2025 19:08:39 +0100
Subject: [PATCH] HACK: Remove inode/directory association

Prevent Baobab being chosen as the default app for directories. Needed
as long as we do not ship a `gnome-mimeapps.list`.
---
 data/org.gnome.baobab.desktop.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/org.gnome.baobab.desktop.in b/data/org.gnome.baobab.desktop.in
index f7c8b432330f..411ffa1587eb 100644
--- a/data/org.gnome.baobab.desktop.in
+++ b/data/org.gnome.baobab.desktop.in
@@ -10,7 +10,6 @@ Icon=org.gnome.baobab
 Terminal=false
 Type=Application
 StartupNotify=true
-MimeType=inode/directory;
 Categories=GTK;GNOME;System;Filesystem;Utility;
 DBusActivatable=true
 X-GNOME-UsesNotifications=true
